### PR TITLE
#5: [Implement UI] adds data retrival logic (closes #5)

### LINKED ISSUE #5: [Implement UI] adds data retrival logic
← #3

## requirements
adds the data retrival logic (commands and queries)

### PR DESCRIPTION
**Pull request by fes300**
Closes #5

### tests performed
I tested that the api showed the correct UI when the api response had no results:
screen shot 2018-07-09 at 15 30 41
when it was fetching data:
screen shot 2018-07-09 at 15 31 35
and when there where results returned:
screen shot 2018-07-09 at 15 35 43

### CODE REVIEW

File: src/components/Form/View.tsx [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
-import { FormFields, FormPlaceholders } from 'model';
-import { FormattedMessage, injectIntl } from 'react-intl';
+import { FetchFields, FormFields, FormPlaceholders } from 'model';
+import { injectIntl } from 'react-intl';
+import * as _ from 'lodash';
 import Input from '@buildo/bento/components/Input';
```
[the remark this answers is not in the document]
Comment (fes300) [reply src/components/Form/View.tsx]: sorry, totally forgot about that.. 😓

**Comment by fes300** [comment]
@ecamellini  I added you as reviewer as suggested by @giogonzo (as he is not fully available today)

File: src/API/API.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { IORestaurantsType } from 'model';
+
+const API_KEY =
+  'xUfPkqmnsgXsHa99dCh5Z4gi-EN4SoqkeMe9ynBuYk5gzjRiTKsZVpu-7q0M8jn0Ho1KGG2Iaf4MmSl8KxWmdDG6QUG8jqhugTD7jeFggJwZbvjZD7XGDTy9eu3gWnYx';
```
Comment (ecamellini) [inline src/API/API.ts]: API key committed in a public repo 😛I would put it in a JSON file that you don't commit here and import it from there (minor, you can also ignore my security rants)

File: src/components/Business/business.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
+@import '~theme/variables.scss';
+
+.business {
+  align-items: stretch;
+  background-color: $mediumGrey;
+  border-radius: 4px;
+  border: solid 1px $darkGrey;
+  flex-shrink: 0;
```
Comment (ecamellini) [inline src/components/Business/business.scss]: When possible, I would move all the layout/disposition stuff to the `JSX` params passed to `View` --> `shrink={false}`

File: src/components/Business/business.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+  img {
+    border-radius: 4px;
+    height: 80px;
+    margin: 16px;
+    width: 80px;
+  }
+
+  .internal_wrapper {
+    align-items: center;
```
Comment (ecamellini) [inline src/components/Business/business.scss]: Same here, you could do all this just with `View` params (`grow`, `hAlignContent`, etc)

File: src/API/API.ts (hunk 2 above)
Comment (fes300) [reply src/API/API.ts]: hahah I know but I found this API_KEY on stack overflow 🙃. I'll do it anyway 👍

**Pull request merged**